src/transformers/pipelines.py

```diff
@@ -2752,6 +3021,18 @@ def _concat_inputs_history(self, inputs: List[List[int]], histories: List[Option
             "model": {"pt": "distilbert-base-cased-distilled-squad", "tf": "distilbert-base-cased-distilled-squad"},
         },
     },
+    "table-question-answering": {
+        "impl": TableQuestionAnsweringPipeline,
+        "pt": AutoModelForTableQuestionAnswering if is_torch_available() else None,
+        "tf": None,
+        "default": {
+            "model": {
+                "pt": "nielsr/tapas-base-finetuned-wtq",
+                "tokenizer": "nielsr/tapas-base-finetuned-wtq",
+                "tf": "nielsr/tapas-base-finetuned-wtq",
```

Comment:
This "tf" is needed as otherwise it would throw the following:

```py
Exception: Impossible to guess which tokenizer to use. Please provided a PretrainedTokenizer class or a path/identifier to a pretrained tokenizer.
```

even when defining the tokenizer or config. I'll slightly refactor the TF/PT interaction in the pipeline so that we may have only framework-specific pipelines, but that'll be for another PR.